kncaputo/underrated
# Feature/movie details

## What's this PR do?
- Completes information and style of movie details section
- Adds `ListItem` component
- Adds `MovieTrailers` component
- Adds `ReactPlayer` and `Carousel` components to format trailers
- Starts responsiveness for MovieDetails views

## Where should the reviewer start?
- Review `ListItem`, `MovieTrailers`, and `MovieDetails` components

## How should this be manually tested?
- Click on a movie in the browser and ensure details show up and trailers can be played

## Any background context you want to provide?
- Responsiveness is only complete for web browser and 1025px screen
- Trailers are only YouTube compatible 
- Testing will need to be added for all components

## Screenshots/GIFs
- <img width="1097" alt="Screen Shot 2020-12-23 at 3 29 11 PM" src="https://user-images.githubusercontent.com/58453566/103041433-a0e5de80-4533-11eb-9f08-4113f4ca6fe9.png">
